src/sphinxcontrib/bibtex/__init__.py

```diff
@@ -43,16 +47,30 @@ def setup(app: Sphinx) -> Dict[str, Any]:
     app.add_directive("bibliography", BibliographyDirective)
     app.add_role("cite", CiteRole())
     app.add_node(bibliography, override=True)
-    app.add_node(raw_latex, latex=(visit_raw_latex, depart_raw_latex),
-                 override=True)
+    app.add_node(raw_latex, latex=(visit_raw_latex, depart_raw_latex), override=True)
     app.add_post_transform(BibliographyTransform)
     app.add_domain(BibtexFootDomain)
     app.add_directive("footbibliography", FootBibliographyDirective)
     app.add_role("footcite", FootCiteRole())
 
+    # Catch bug in newer docutils
+    if (
+        (0, 18, 0, "final", 0, True)
+        <= docutils.__version_info__
+        < (0, 20, 0, "final", 0, True)
```

Comment:
Please use tuple comparison against just the numeric part of the version; e.g. ``(0, 18) <= doctutils.__version_info__ < (0, 20)`` should do the trick.